examples/list.rs

```diff
@@ -1,24 +1,70 @@
-use std::{
-    error::Error,
-    io,
-    time::{Duration, Instant},
-};
+use std::{error::Error, io};
 
 use crossterm::{
     event::{self, DisableMouseCapture, EnableMouseCapture, Event, KeyCode, KeyEventKind},
     execute,
     terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
 };
-use ratatui::{prelude::*, widgets::*};
+use ratatui::{prelude::*, style::palette::tailwind, widgets::*};
+
+const TODO_HEADER_BG: Color = tailwind::BLUE.c950;
```

Comment:
using consts for these are good, but perhaps it's worth moving them closer to the impl Widget block? Not 100% sure on this - thoughts?

Reply:
I like the idea of putting const at the top but ofc I'm open to suggestions. I think putting const closer to impl Widget block would make that area look crowded.

Reply:
I like having consts at the top too, so its like easy-to-change parameters.